apkid/rules/dex/packers.yara

```diff
@@ -70,3 +70,25 @@ rule medusah_appsolid_dex : packer
   condition:
     is_dex and $loader and $main_activity
 }
+
+rule apkguard : packer
+{
+  meta:
+    description = "ApkGuard"
+    info = "http://apkguard.io/"
+    // "Base64.decode("UEsDBB..."
+
+  strings:
+    $method = "attachBaseContext"
+    $dexclassloader = "DexClassLoader"
+    $getclass = "getClass"
+    $getdeclaredmethod = "getDeclaredMethod"
+    $import = "java/lang/reflect/InvocationTargetException"
+    $getfilesdir = "getFilesDir"
+    $dex = "UEsDBB"
+    $b64decode = "Base64"
+
+  condition:
+    is_dex and
+    all of them
+}
```

Comment:
Nitpick, but could you always end files with a newline?

Reply:
Of course from now on. Np